component/portal/src/main/java/org/exoplatform/portal/mop/rest/NavigationRest.java

```diff
@@ -247,15 +258,15 @@ private Response getNavigations(HttpServletRequest request, // NOSONAR
         UserNode rootNode = userPortal.getNode(navigation, scope, userFilterConfig, null);
         nodes = rootNode.getChildren();
       }
-      List<ResultUserNode> resultNodes = convertNodes(nodes);
+      List<ResultUserNode> resultNodes = convertNodes(nodes, userIdentity, expandPageDetails);
       return Response.ok(resultNodes).build();
     } catch (Exception e) {
       LOG.error("Error retrieving ");
       return Response.status(500).build();
     }
   }
 
-  private List<ResultUserNode> convertNodes(Collection<UserNode> nodes) {
+  private List<ResultUserNode> convertNodes(Collection<UserNode> nodes, org.exoplatform.services.security.Identity identity, boolean expandPageDetails) {
```

Comment:
```suggestion
  private List<ResultUserNode> convertNodes(Collection<UserNode> nodes, Identity identity, boolean expandPageDetails) {
```

since org.exoplatform.services.security.Identity is already imported